### PR TITLE
sticky focused marking

### PR DESCRIPTION
**Pull request by AndiBraimllari**

### CODE REVIEW

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -408,7 +408,7 @@ <h5 class="card-title">
                 </div>
             </div>
             <ul class="list-group list-group-flush listStyling">
-                <div id="spinnerBorder" class="spinner-border" role="status">
+                <div class="spinner-border" role="status">
```
Comment (AndiBraimllari) [inline src/app/home/home.component.html:411]: Any idea if class 'spinner-border' was used anywhere @Kledis98 ?

Comment (AndiBraimllari) [reply src/app/home/home.component.html:411]: I dropped the ID and utilized the class for the styling.

Comment (Kledis98) [reply src/app/home/home.component.html:414]: It is all right. It was used only there, indicating the loading state.

**Review by Kledis98 — COMMENTED** [review]
Great!

**Pull request merged**